integration-cli/docker_cli_rename_test.go

```diff
@@ -97,7 +97,7 @@ func TestRenameCheckNames(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	logDone("rename - running container")
+	logDone("rename - check the first container's status")
```

Comment:
Yes, my problem is the log message you have has no meaning to me, I'm not sure what's being checked from reading the log message.